Agreements/Internal/Nor_S29.txt

```diff
@@ -0,0 +1,3 @@
+COPX:EGCC:*:POL:*:*:North:Sector 29:19000:*:POL
+COPX:EGGP:*:POL:*:*:North:Sector 29:19000:*:POL
+COPX:EGNR:*:POL:*:*:North:Sector 29:19000:*:POL
```

Comment:
These changes have been made _after_ the climbing/descending arrows changes were committed, hence why these aren't all ^POL... think it might be best to figure those changes out before this is merged.